app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
Apparently this is the first model created after moving this repo to Rails 5. All models should be descending from this abstract class now. I can update the others in this PR as well or create a new repo.

Reply:
@rsl yah, let's update the models to inherit from ApplicationRecord.  It is so strange, the app started on Rails 5...but I didn't realize this ApplicationRecord inheritance change until a couple days ago, and even then, I assumed it was in a Rails 5 version bump....I'm guessing it never came up since I don't use 'rails generate model'.....anyway, strange.
If you could update the inheritance for the models in this PR, that would be amazing!!!!  Thank you!!!

Reply:
yeah i ran rails g model for this so it triggered it. wanted it to create all the spec/factory files.